tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- needs to contain all frameworks which src projects wish to restore -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.0.0-preview-27625-16"/>
```

Comment:
(nit) you can use the latest 1.0.0-preview build: `1.0.0-preview-27630-5`. We just spun it and will have the official `1.0.0` up soon.

Reply:
Will update to the new build and change to 1.0.0 as soon as it is available.

---
In reply to: [279941849](https://github.com/dotnet/machinelearning/pull/3623#discussion_r279941849) [](ancestors = 279941849)

Reply:
It's available on myget now. You should be able to use `1.0.0`.